src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const employeeList = [...document.querySelectorAll('tr')].slice(1, -1);
+const mainList = document.querySelector('tbody');
+const menu = document.querySelector('thead').firstElementChild.children;
```

Comment:
with event delegation `thead` will be enough

Reply:
done